internal/kafka/internal/services/kafka.go

```diff
@@ -619,24 +625,25 @@ func (k *kafkaService) DeprovisionKafkaForUsers(users []string) *errors.ServiceE
 	return nil
 }
 
-func (k *kafkaService) DeprovisionExpiredKafkas() *errors.ServiceError {
-	dbConn := k.connectionFactory.New().Model(&dbapi.KafkaRequest{}).Session(&gorm.Session{})
-
-	var typesWithLifespan []string
-	for _, kafkaInstanceType := range k.kafkaConfig.SupportedInstanceTypes.Configuration.SupportedKafkaInstanceTypes {
-		if kafkaInstanceType.HasAnInstanceSizeWithLifespan() {
-			typesWithLifespan = append(typesWithLifespan, kafkaInstanceType.Id)
-		}
+func (k *kafkaService) kafkaWithExpiredAtShouldBeDeprovisioned(kafkaRequest *dbapi.KafkaRequest, currentTime time.Time) (bool, error) {
+	glog.V(10).Infof("Evaluating expiration time of kafka request '%s' with instance type '%s', size ID '%s' and status '%s'", kafkaRequest.ID, kafkaRequest.InstanceType, kafkaRequest.SizeId, kafkaRequest.ID, kafkaRequest.Status)
+	shouldDeprovision := false
+	if currentTime.After(kafkaRequest.ExpiresAt.Time) {
+		glog.V(10).Infof("Kafka ID '%s' has expired", kafkaRequest.ID)
+		shouldDeprovision = true
```

Comment:
```suggestion
		return true, nil
```

Reply:
:+1: . This was a leftover of a refactoring. I will change it

Reply:
updated